### ISSUE TITLE
False positive error "A second item with name '_' imported. Try to use an alias."

### ISSUE THREAD

**Issue by WaffleLapkin** [intellij-rust/intellij-rust#5832]
## Environment

* **IntelliJ Rust plugin version:** 0.3.128.3254-201-nightly
* **Rust toolchain version:** 1.44.1 (c7087fe00 2020-06-17) x86_64-unknown-linux-gnu
* **IDE name and version:** CLion 2020.1.2 (CL-201.7846.88)
* **Operating system:** Linux 5.7.7-arch1-1

## Problem description

When at least 2 items are imported as `_` idea reports error "`A second item with name '_' imported. Try to use an alias. [E0252]`"

## Steps to reproduce

Write 2 `use`s with `as _`, e.g.:
```rust
use core::fmt::{Debug as _, Display as _};
```
![image](https://user-images.githubusercontent.com/38225716/88941784-1b1a7b80-d292-11ea-9573-970239badff7.png)

**Comment by Undin** [comment]
Duplicate of #5832

**Comment by Undin** [comment]
Duplicate of #5067
